### ISSUE TITLE
Montblanc dtype option ignored

### ISSUE THREAD

**Issue by JSKenyon** [ratt-ru/CubiCal#223]
Somehow specifying double in the parset never reaches Montblanc. Need to be fixed.

**Comment by JSKenyon** [comment]
I have volunteered @o-smirnov to fix this given that he removed the functionality. Montblanc and MeqTrees differ at the 10e-5 level when using float in Montblanc (precision not configurable in MeqTrees). I am in absolute agreement that the default should be float, but I would like the option of doing the predict in double. That way I will know that when I simulate observed data in MeqTrees but do the model predict with Montblanc that I am comparing apples with apples.

**Comment by o-smirnov** [comment]
There was some good reason to remove it, but the margin was too small to write it out... let me think...

**Comment by o-smirnov** [comment]
Yeah, I remember now, having to support both float and double models made the auto-rebinning logic annoyingly arcane... so, what would be the least traumatic way to reinstate it?

**Comment by JSKenyon** [comment]
I wouldn't mind if using double was mutually exclusive with rebinning. Rebinning constitutes a reduction in precision anyway. So perhaps reinstate the option but make it shout if rebinning is enabled and double is specified?
